### PR TITLE
Packaging Up DQC

### PR DESCRIPTION
**Pull request by chrislococo-wf**
##### Pull request for: [XS-2631](https://jira.webfilings.com/browse/XS-2631)

[XS-2631](https://jira.webfilings.com/browse/XS-2631) - Make dqc_us_rules pip installable with a setup.py
- We need to make DQC pip installable. 
#### Changes:

Enter some description here...
- Added the standard windowdressing for a pip library and configured it for dqc_us_rules

Please review: @jasonleinbach-wf @mindeerickson-wf @andrewperkins-wf @brianmyers-wf @bradbenjamin-wf @derekgengenbacher-wf

### CODE REVIEW

File: dqc_us_rules/_pkg_meta.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""
+Module Docstring
```
Comment (brianmyers-wf) [inline dqc_us_rules/_pkg_meta.py]: This isn't a very informative docstring... is it necessary? or is there a better description?

**Comment by brianmyers-wf** [comment]
One minor comment but otherwise looks great.

File: dqc_us_rules/_pkg_meta.py (hunk 1 above)
Comment (chrislococo-wf) [inline dqc_us_rules/_pkg_meta.py]: Nope, it is an auto docstring, damnit.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Tested with Chris to verify all expected files are in the new dqc_us_rules folder.  

QA +1

**Comment by andrewperkins-wf** [comment]
+1 approved for merge

**Pull request merged**